index.js

```diff
@@ -0,0 +1,109 @@
+// document.addEventListener("DOMContentLoaded", () => {
+//   let movieOption;
+//   let movieArray
+//   let movieInfoRequest
+
+//   const apiRequest = async () => {
+//     try {
+//       movieInfoRequest = await axios.get("https://ghibliapi.herokuapp.com/films");
+//       movieArray = movieInfoRequest.data;
+//     } catch (error) {
+//       console.log(error)
+//     }
+//   }
+  
+
+//   const populateSelected = async () => {
+//     apiRequest()
+//     for (let i = 0; i <= movieArray.length; i++) {
+//       console.log(movieArray[i])
+//       // let select = document.querySelector("#movieTitles");
+//       // let movieTitlesOption = document.createElement("option");
+//       // movieTitlesOption.value = movieArray[i].title;
+//       // movieTitlesOption.innerText = movieArray[i].title;
+//       // select.appendChild(movieTitlesOption);
+      
+//     }
+//   }
+//   populateSelected()
+
+// ;
+// })
+
+document.addEventListener("DOMContentLoaded", () => {
+  let form = document.querySelector("form");
+  let movieInfo = document.querySelector(".movieInfo");
+  let select = document.querySelector("#movieTitles");
+  let options;
+  let movies;
+  let userSelected;
+
+  const requestMovieInfo = async () => {
+    try {
+      movies = await axios.get(`https://ghibliapi.herokuapp.com/films`)
+      
+      movieOption = movies.data
+      
+    } catch(error) {
+      console.log(error)
+    }
+    
+    const selectMovieOptions = async () => {
+      
+      for (let i = 0; i <= movieOption.length; i++) {
+        options = document.createElement("option");
+        options.innerText = movieOption[i].title;
+        // options.value = movieOption[i].title;
+        
+        select.appendChild(options)
+        
+      }
+
+    }
+    selectMovieOptions()
+        
+
+
+    document.addEventListener("input" , (event) => {
+      if (event.target.id !== "movieTitles") return;
+      userSelected = event.target.value;
+      movieInfo.innerHTML = "";
+      for (let i = 0; i <= movieOption.length; i++) {
```

Comment:
This is pretty nice, if you look closer at what you have here is a filter. You created your own filter, which is awesome, but next time I would be nice to see you use the filter array method